velox/functions/prestosql/window/LeadLag.cpp

```diff
@@ -87,6 +87,13 @@ class LeadLagFunction : public exec::WindowFunction {
   }
 
  private:
+  // Row number used to denote the specified row should be set to default
```

Comment:
Nit - rewrite

Lead/Lag return default value (using kDefaultValueRow) if offsets for target rowNumbers are outside the partition. If offset is null, then the functions return null (using kNullRow) . 

kDefaultValueRow needs to be a negative number so that WindowPartition::extractColumn calls skip this row. It is set to -2 to distinguish it from kNullRow which is -1.